state/config.go

```diff
@@ -1,22 +1,22 @@
 package state
 
 import (
-	luraconfig "github.com/luraproject/lura/v2/config"
-
 	"github.com/krakend/krakend-otel/config"
+	kotelconfig "github.com/krakend/krakend-otel/config"
+	luraconfig "github.com/luraproject/lura/v2/config"
 )
 
 type Config interface {
 	OTEL() OTEL
-	GlobalOpts() *config.GlobalOpts
+	GlobalOpts() config.GlobalOpts
```

Comment:
I'm not sure why they were returning a pointer to the configuration (which is a global variable as far as I could see). If you do that you risk updating it dynamically and potentially breaking it. Not sure why you would want to do that, but perhaps they had some use case in mind.
I changed this also because I'm now merging the global static labels with the local ones and such a change doesn't apply globally.